lib/src/signed_video_h26x_sign.c

```diff
@@ -373,6 +373,8 @@ generate_sei_nalu(signed_video_t *self, uint8_t **payload, uint8_t **payload_sig
 
     // End of GOP. Reset flag to get new reference.
     self->gop_info->has_reference_hash = false;
+    // Reset the timestamp to not include it in the next SEI.
```

Comment:
A little weird formulation. How about "Reset the timestamp to avoid including a duplicate in the next SEI."